templates/test/ci/prow-aks-aso/patches/aks-pool2.yaml

```diff
@@ -35,5 +35,5 @@ spec:
         name: ${CLUSTER_NAME}
       mode: User
       type: VirtualMachineScaleSets
-      vmSize: "${AZURE_AKS_NODE_MACHINE_TYPE:=Standard_D2s_v3}"
+      vmSize: "${AZURE_AKS_NODE_MACHINE_TYPE:=Standard_D2ds_v5}"
```

Comment:
Is there also a newer version of Standard_B2s we could add to the self-managed templates if we're still seeing capacity issues for those too?